wiki/Reporting_Bad_Behaviour/en.md

```diff
@@ -16,39 +12,49 @@ It is not only limited to in-game chat but also in other places such as the foru
 
 ### Using the !report command in-game
 
-Usage: `!report username [reason]`
+Usage: `!report user_name [reason]`
```

Comment:
```suggestion
Usage: `!report user_name [reason]`
```
I think you can just keep the username without ( _ ) underscore since in the next paragraph, all space letters has been explained, so i think you don't need to use user_name with underscore here.